scripts/metricsLib/oss_metric_entities.py

```diff
@@ -145,23 +146,39 @@ def __init__(self, repo_git_url):
 
         self.repo_owner = owner
 
-        endpoint = f"{AUGUR_HOST}/owner/{self.repo_owner}/repo/{repo_name}"
+        endpoint = f"{AUGUR_HOST}/repos"
         super().__init__(repo_name,endpoint)
 
-        try:
-            response = requests.post(
-                self.augur_util_endpoint, timeout=TIMEOUT_IN_SECONDS)
-            response_json = json.loads(response.text)
-        except Exception:
-            reponse_dict = {}
+        
+        response = requests.get(
+            self.augur_util_endpoint, timeout=TIMEOUT_IN_SECONDS)
+        response_json = json.loads(response.text)
+        #print(response_json)
 
         try:
-            self.repo_id = response_json[0]["repo_id"]
-            self.repo_group_id = response_json[0]["repo_group_id"]
+            len(response_json)
+            repo_val = next(x for x in response_json if x['repo_name'] == repo_name and x['rg_name'] == owner)
```

Comment:
I think there's a typo here with "rg_name"? Is it supposed to be "org_name" or there's no typo and rg stands for repo group?

Reply:
'rg' stands for repo group. It's just what augur calls it on this endpoint for some reason.